### PR TITLE
make tachyon version configurable

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@shivaram How about this one?

File: tachyon/init.sh [hunk 1]
```diff
@@ -7,8 +7,6 @@ if [ -d "tachyon" ]; then
   return 0
 fi
 
-TACHYON_VERSION=0.5.0
-
 # Github tag:
 if [[ "$TACHYON_VERSION" == *\|* ]]
 then
```
Comment (shivaram) [inline tachyon/init.sh:12]: Could you add a print message here something like: `echo "Tachyon git hashes are not yet supported. Please specify a Tachyon release version."`

**Comment by ghost** [comment]
@shivaram updated.

**Comment by shivaram** [comment]
LGTM

**Pull request merged**